packages/next/build/webpack-config.ts

```diff
@@ -1277,6 +1277,16 @@ export default async function getBaseWebpackConfig(
       ].filter(Boolean),
     },
     plugins: [
+      // Produce source maps for middlewares
+      new webpack.SourceMapDevToolPlugin({
```

Comment:
Just to be super clear about this @sokra this **does not** enable sourcemap generation for all files and then discard only the ones in `include` right?

Reply:
This should probably only be applied for production right?

Reply:
It only generates SourceMaps for the included files, *but* in enables SourceMap generation in loaders for all modules, so that could affect performance...